README.md

```diff
@@ -25,6 +25,58 @@ yarn install  # installs dependencies
 yarn run dev  # runs the webserver and backend (for online multiplayer games)
 ```
 
+## Build docker
+
+To build an docker image all dependencies must be installed and installed before the docker build process. Run the gameserver  once locally to compile. Shut down the server with CTRL-C as soon as it is up and running.
+
+```bash
+yarn install && yarn run dev
+```
+
+First build a the common `fbg-common` docker image.
+
+```bash
+docker build -t fbg-common:latest ./common
+```
+
+Now the images from the compose file can be build.
+
+```bash
+docker-compose build
+```
+
+The previous steps can be also called by a setup script:
+
+```bash
+./docker-build.sh
+```
+
+
+
+After the build process the server can be started via docker compose.
+
+```bash
+docker-compose up -d
+```
+
+Usage:
+
+```bash
+Usage: ./docker-build.sh -[d|b|e|i|r|h]
+
+   -d,    Installs dependencies and run the webserver and backend.
+   -b,    Build docker image
+   -e,    Export build docker images
+   -i,    Import docker images
+   -r,    Prune docker images
+   -h,    Print this help text
+
+If the script will be called without parameters, it will run:
+    ./docker-build.sh -d -b
+```
```

Comment:
Can you move this to a new story on web/docs? That's where most of our technical documentation is, the README is focused on the main commands for beginners only.

Reply:
sure - bu I'm not very familiar with storys so I just did copy/paste now from the others.